static/app/utils/profiling/renderers/uiFramesRenderer.tsx

```diff
@@ -143,27 +110,14 @@ class UIFramesRenderer {
       this.positions[positionOffset + 10] = x2;
       this.positions[positionOffset + 11] = y2;
 
-      // @TODO check if we can pack bounds across vertex calls,
-      // we are allocating 6x the amount of memory here
-      const boundsOffset = index * VERTICES * BOUNDS;
-
-      for (let i = 0; i < VERTICES; i++) {
-        const offset = boundsOffset + i * BOUNDS;
-
-        this.bounds[offset] = x1;
-        this.bounds[offset + 1] = y1;
-        this.bounds[offset + 2] = x2;
-        this.bounds[offset + 3] = y2;
-      }
-
-      for (let i = 0; i < VERTICES; i++) {
-        const offset = boundsOffset + i * BOUNDS;
-
-        this.bounds[offset] = x1;
-        this.bounds[offset + 1] = y1;
-        this.bounds[offset + 2] = x2;
-        this.bounds[offset + 3] = y2;
-      }
+      const type = frame.type === 'frozen' ? 1 : 0;
+      const typeOffset = index * 6;
```

Comment:
This 6 refers to the vertices right? And that's the same as `VERTICES_PER_FRAME` right?
```suggestion
      const typeOffset = index * VERTICES_PER_FRAME;
```